tests/factories/user.js

```diff
@@ -16,6 +16,8 @@ FactoryGuy.define('user',{
     id:        FactoryGuy.generate('id'),
     firstName: FactoryGuy.generate('collectionFirstName'),
     lastName:  FactoryGuy.generate('collectionLastName'),
+    email: "rock.programmer@hotmail.com",
```

Comment:
We can use faker for creating data in future may be like we do in api.

https://github.com/johno/ember-faker

We don't need to do this in PR.

Adding this comment just for our attention. :)

CC: @steveyken  @Darshan-Chauhan  @patrixr